examples/node/MarketDataSubscription.js

```diff
@@ -42,16 +42,19 @@ socket.on('err', function (data) {
     console.log(data);
 });
 
-socket.on('subscribed', function () {
+socket.on('subscribed', function (correlationIds) {
     console.log('Subscribed');
+    console.log('Correlation Ids: ' + correlationIds);
 });
 
-socket.on('unsubscribed', function () {
+socket.on('unsubscribed', function (correlationIds) {
     console.log('Unsubscribed');
+    console.log('Correlation Ids: ' + correlationIds);
 });
 
-socket.on('unsubscribed all', function () {
+socket.on('unsubscribed all', function (correlationIds) {
```

Comment:
I agree, `unsubscribed all` is redundant and error prone, the client has to remember that he has to listen to 2 different 'unsubscribed' events.